mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -158,6 +175,413 @@ properties:
+                      name: 'groupIndexes'
+                      description: |
+                        The index of the submatch to extract as findings. When not specified, the entire match is returned. No more than 3 may be included.
+                      item_type: Api::Type::Integer
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'dictionary'
+                  description: Dictionary which defines the rule.
+                  immutable: true
+                  properties:
+                    - !ruby/object:Api::Type::NestedObject
+                      name: 'wordList'
+                      description: List of words or phrases to search for.
+                      properties:
+                        - !ruby/object:Api::Type::Array
+                          name: 'words'
+                          required: true
+                          description: |
+                            Words or phrases defining the dictionary. The dictionary must contain at least one
+                            phrase and every phrase must contain at least 2 characters that are letters or digits.
+                          item_type: Api::Type::String
+                    - !ruby/object:Api::Type::NestedObject
+                      name: 'cloudStoragePath'
+                      description: Newline-delimited file of words in Cloud Storage. Only
+                        a single file is accepted.
+                      properties:
+                        - !ruby/object:Api::Type::String
+                          name: 'path'
+                          required: true
+                          description: |
+                            A url representing a file or path (no wildcards) in Cloud Storage. Example: `gs://[BUCKET_NAME]/dictionary.txt`
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'storedType'
+                  description: A reference to a StoredInfoType to use with scanning.
+                  immutable: true
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'name'
+                      required: true
+                      description: |
+                        Resource name of the requested StoredInfoType, for example `organizations/433245324/storedInfoTypes/432452342`
+                        or `projects/project-id/storedInfoTypes/432452342`.                
+                    - !ruby/object:Api::Type::String
+                      name: 'createTime'
+                      description: |
+                        The creation timestamp of an inspectTemplate. Set by the server.
+                      output: true
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'surrogateType'
+                  description: |
+                    Message for detecting output from deidentification transformations that support reversing.
+                  immutable: true
```

Comment:
I've checked this field and it runs fine without `ForceNew`. Hence, removing `immutable:true`.